### PR TITLE
Set up hapi server

### PR DESCRIPTION
**Pull request by shiryz**

### CODE REVIEW

**Comment by des-des** [comment]
@shiryz pull has no description

**Comment by des-des** [comment]
@shiryz This closing a pull and making a new one is not ideal. I am now reviewing code that I have already looked at.

**Comment by shiryz** [comment]
@des-des , sorry got confusing because thought I should merge it to a different branch not master!

**Comment by shiryz** [comment]
@des-des , Added 2 tests for now because we don't have routes.

File: tests/server-test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+tape('Check the / route', (t) => {
+  var options = {
+    method: 'GET',
+    url: '/'
+  }
+  server.inject(options, (response) => {
+    t.equals(response.statusCode, 200, 'status code is 200');
+    var servedFile = response.result.toString();
+    var file = fs.readFileSync(path.join(__dirname, '../public/html/landing-page.html')).toString();
+    t.equal(servedFile, file, 'payload contains no content');
```
Comment (des-des) [inline tests/server-test.js]: wikid

File: tests/server-test.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  });
+});
+
+tape('Check the / route', (t) => {
+  var options = {
+    method: 'GET',
+    url: '/'
+  }
+  server.inject(options, (response) => {
+    t.equals(response.statusCode, 200, 'status code is 200');
```
Comment (des-des) [inline tests/server-test.js]: wikid

File: src/server.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+const fs = require('fs');
+
+const server = new Hapi.Server();
+
+var tls = {
+    key : fs.readFileSync('./key.pem'),
+    cert : fs.readFileSync('./cert.pem')
+};
+
+server.connection({
```
Comment (des-des) [inline src/server.js]: FYI
```js
  var connectionOpts = process.env === 'production'
    ? { /*produciton / heroku config */}
    : { /* dev config*/ }
```

**Pull request merged**